rampsdk/src/main/java/network/ramp/sdk/events/model/Payloads.kt

```diff
@@ -65,11 +59,9 @@ data class Purchase(
     val asset: Asset, // description of the purchased asset (address, symbol, name, decimals)
     val receiverAddress: String, // blockchain address of the buyer
     val assetExchangeRate: Double,// price of 1 whole token of purchased asset, in fiatCurrency
-    @Deprecated("This parameter will be removed in future")
-    val purchaseViewToken: String,
     val status: String, // purchase status
     val paymentMethodType: String, // type of payment method used to pay for the swap: 'MANUAL_BANK_TRANSFER' | 'AUTO_BANK_TRANSFER' | 'CARD_PAYMENT' | 'APPLE_PAY'
-    val finalTxHash: String? = null // hash of the crypto transfer blockchain transaction, filled once available
```

Comment:
It's received later in the flow, so the first event probably won't contain this property.
I'll leave it at it was